app/src/test/java/com/github/se/signify/model/home/quest/FirestoreQuestRepositoryTest.kt

```diff
@@ -285,4 +285,10 @@ class FirestoreQuestRepositoryTest {
     verify(mockTask).addOnCompleteListener(captor.capture())
     captor.value.onComplete(mockTask)
   }
+
+  @Test
+  fun fetchVideo_constructsCorrectPath() {
```

Comment:
According to my previous comment, it's more secure to add tests covering the failure cases. 
So we can test scenarios where the videoPath in a Quest is empty, invalid, or inaccessible.